guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -33,83 +33,97 @@ Optional: If the *Registration* feature is not enabled on your {SmartProxy}, ent
 
 .Procedure
 . Navigate to *Hosts* > *Register Host*.
+. Optional: Select a different *Organization*.
+. Optional: Select a different *Location*.
 . Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: Operating System, Activation Key(s) and Lifecycle environment.
-. From the *Operating System* list, select the operating system of hosts that you want to register.
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through. You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
-. Optional: *Insecure* - This makes the first call insecure, however, during this first call, hosts download the CA file from {Project}. Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+Fields that inherit value from *Host group*: Operating system, Activation Key(s) and Lifecycle environment.
+. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+ifdef::satellite[]
+You must select the internal {SmartProxy} if you do not want to use an external {SmartProxy}.
+endif::[]
+. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
+ifdef::satellite[]
+Specifying an operating system is required when you register machines without subscription-manager, such as Debian or Ubuntu.
+endif::[]
+. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+During this first call, hosts download the CA file from {Project}.
+Hosts will use this CA file to connect to {Project} with all future calls making them secure.
 +
 If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
 Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
 +
-
-. From the *Setup REX* list, select whether you want to deploy {Project} SSH keys to hosts or not.
+The following is an example of the `curl` command with the `--insecure` option:
 +
-If set to `Yes`, SSH keys will be installed on the registered host. The inherited value is based on the `host_registration_remote_execution` parameter.
-It can be inherited e.g. from host group, operating system, organization. When overridden, the selected value will be stored on host parameter level.
+[options="nowrap", subs="+quotes,attributes"]
+----
+curl -sS --insecure https://{foreman-example-com}/register...
```

Comment:
I guess this needs some input from @stejskalleos, my wild guess is `/unattended/register` but not sure about the rest.

Also note we should not use `-k` or `--insecure` in our docs, we should always ask our users to install CA certificate. I think this could be modularized too:

```
guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc
77:# update-ca-trust enable
78:# update-ca-trust

guides/doc-Administering_Red_Hat_Satellite/topics/Configuring_Identity_Management.adoc
226:# update-ca-trust enable
227:# update-ca-trust

guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc
112:# update-ca-trust enable
113:# update-ca-trust

guides/doc-Provisioning_Guide/topics/Infoblox_Integration.adoc
40:# update-ca-trust enable
43:# update-ca-trust extract

guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc
374:# update-ca-trust enable
391:# update-ca-trust extract
```

Reply:
@stejskalleos Can you please clarify what the proper example URL should look like?

Reply:
the URL is `https://foreman.example.com/register`